### PR TITLE
CCD-3853 run FTAs against next test definition release

### PR DESCRIPTION
**Pull request by mattnayler**
### JIRA link (if applicable) ###

   [CCD-3853](https://tools.hmcts.net/jira/browse/CCD-3853) _"Intermittent Data Store FT failures in nightly build pipelines"_

### Change description ###

Run FTAs against the next `ccd-test-definitions` release, see hmcts/ccd-test-definitions#58.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/ccd-definition-store-api/values.preview.template.yaml [hunk 1]
```diff
@@ -23,7 +23,8 @@ java:
     # enable whenever required and provide host url to match with corresponding data-store-api
     ELASTIC_SEARCH_ENABLED: true
 
-    ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-1260-es-master
+    # TODO: remove TEMPORARY OVERRIDE to test ElasticSearch, reinstate: ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-1260-es-master
+    ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-2138-es-master
```
Comment (mattnayler) [inline charts/ccd-definition-store-api/values.preview.template.yaml:27]: This temporary change should be reversed before merging.

File: build.gradle [hunk 2]
```diff
@@ -286,7 +286,7 @@ subprojects { subproject ->
         testImplementation group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testImplementation group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
 
-        testImplementation group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.4'
+        testImplementation group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.6-CCD-3853'
```
Comment (mattnayler) [inline build.gradle:289]: Should be updated with proper release number once hmcts/ccd-test-definitions#58 is released.

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

File: build.gradle (hunk 2 above)
Comment (mattnayler) [reply build.gradle]: Resolved in 5eaf89e162d49dc8207e4d20aad21b64aed21ea4

File: charts/ccd-definition-store-api/values.preview.template.yaml (hunk 1 above)
Comment (mattnayler) [reply charts/ccd-definition-store-api/values.preview.template.yaml]: Resolved in b5c847d946701b54984ad3704f88bffd5f9ac567.

**Comment by mattnayler** [comment]
This PR is now ready for release: after removing temporary pipeline configuration.

**Pull request merged**